sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -195,16 +195,21 @@ case class AlterTableRenameCommand(
       DDLUtils.verifyAlterTableType(catalog, table, isView)
       // If an exception is thrown here we can just assume the table is uncached;
       // this can happen with Hive tables when the underlying catalog is in-memory.
-      val wasCached = Try(sparkSession.catalog.isCached(oldName.unquotedString)).getOrElse(false)
```

Comment:
The existing implementation uses `Catalog` APIs (`isCached`), whereas this PR uses `CacheManager` directly. If this approach is not desired, we can update `Catalog` API to expose `StorageLevel`.